lib/taskwarrior-web/model/task.rb

```diff
@@ -136,6 +136,17 @@ def self.respond_to?(method_sym, include_private = false)
       end
     end
 
+    ################################################
+    # CLASS METHOD FOR SYNCRONISING WITH TASK SERVER
+    ################################################
+
+    ##
+    # Syncronise the local task database with the server
+    # TODO: These args are going to be ignored!
+    def self.sync(*args)
```

Comment:
I'm not sure where the best place for this method is, really, but don't think it belongs on the `Task` model. Perhaps on the main `TaskwarriorWeb` module?